sled-agent/src/rack_setup/plan/service.rs

```diff
@@ -362,6 +373,61 @@ impl Plan {
                 })
             }
 
+            // All sleds get an NTP server, but the first few are nominated as
+            // boundary servers, responsible for communicating with the external
+            // network.
+            {
+                let id = Uuid::new_v4();
+                let address = addr_alloc.next().expect("Not enough addrs");
+                let zone = dns_builder.host_zone(id, address).unwrap();
+
+                let (services, svcname) = if idx < BOUNDARY_NTP_COUNT {
+                    boundary_ntp_servers
+                        .push(format!("{}.host.{}", id, DNS_ZONE));
+                    (
+                        // XXXNTP - these boundary servers need a path to the
+                        // external network via OPTE.
```

Comment:
PSA to @luqmana - we should coordinate this integration with https://github.com/oxidecomputer/omicron/pull/2419